src/Controller/UsersController.ts

```diff
@@ -49,19 +51,37 @@ export class UsersController extends BaseHttpController {
     return this.json(updateResult.authResponse)
   }
 
-  @httpGet('/:userId/settings')
+  @httpGet('/:userUuid/settings')
   async getSettings(request: Request, response: Response): Promise<results.JsonResult> {
-    if (request.params.userId !== response.locals.user.uuid) {
+    if (request.params.userUuid !== response.locals.user.uuid) {
       return this.json({
         error: {
           message: 'Operation not allowed.'
         }
       }, 401)
     }
 
-    const { userId } = request.params
-    const result = await this.doGetSettings.execute({ userUuid: userId })
+    const { userUuid } = request.params
+    const result = await this.doGetSettings.execute({ userUuid })
 
     return this.json(result)
   }
+
+  @httpGet('/:userUuid/settings/:settingName')
+  async getSetting(request: Request, response: Response): Promise<results.JsonResult> {
+    if (request.params.userUuid !== response.locals.user.uuid) {
+      return this.json({
+        error: {
+          message: 'Operation not allowed.'
+        }
+      }, 401)
+    }
+
+    const { userUuid, settingName } = request.params
+    const result = await this.doGetSetting.execute({ userUuid, settingName })
+
+    if (result.success) return this.json(result)
```

Comment:
can we make the if styling consistent? Meaning that if/else blocks should have opening and closing brackets instead of inline ... by experience I found it easy to miss if they sometimes are inline and sometimes full brackets.

This setting should also be added to linter - separate issue

Reply:
Sure, will do.

Just to justify, my rule is: omit the brackets if the body of the if or else starts on the same line. Never did me wrong in any language.

Probably this linter rule will be necessary to ensure this is not done out of habit.

Reply:
Ever since that infamous (heartbleed?) bug that was caused by missing brackets in an if/else I've always started using brackets 😅

Reply:
Will merge this and fix this and other brackets I can find in the next PR.

Reply:
@mobitar oh I know about that issue, encountered it too once on a 24 h gamedev competition -- lost quite a bit of time debugging

but the rule I use is not susceptible to that -- you ALWAYS put the single expression on the same line as the keyword
this way you can never mistake a single-expression if with braces for a single-expression if without braces (and start adding next expressions/statements below) -- there is no misleading indentation or anything like that